api/room/schema_query.py

```diff
@@ -241,6 +249,15 @@ def resolve_all_remote_rooms(self, info, return_all=None):
                 break
         return AllRemoteRooms(rooms=remote_rooms)
 
+    @Auth.user_roles('Admin', 'Default User')
```

Comment:
According to the story description, an admin should be the only one to access this. Kindly changes this.

Reply:
Alright noted. I will sort this